### PR TITLE
Release 1.1.1

### PR DESCRIPTION
**Pull request by i7an**
## Motivation
Release a new version with [the patch](https://github.com/railsware/actionmailer-balancer/pull/28).

## Changes

- Updated Gemfile
- Added Gemfile.lock to gitignore
- Bumped version

## How to test

N/A

## Images and GIFs

N/A

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,6 +6,8 @@
 /pkg/
 /spec/reports/
 /tmp/
+Gemfile.lock
```
Comment (mklocek) [inline .gitignore:9]: Can you elaborate on this decision? If we don't have lock, our tests may not be stable in the future (not all dependencies have pinned versions)

Comment (i7an) [reply .gitignore:9]: https://yehudakatz.com/2010/12/16/clarifying-the-roles-of-the-gemspec-and-gemfile/
> Do not check your Gemfile.lock into version control

Comment (i7an) [reply .gitignore:9]: Though it might be outdated. Will take a closer look tomorrow.
https://github.com/rubygems/rubygems/issues/3372

Comment (i7an) [reply .gitignore:9]: Yeah bundler recommends checking it in.
https://bundler.io/v2.5/man/bundle-install.1.html#THE-GEMFILE-LOCK
> As a result, you SHOULD check your Gemfile.lock into version control, in both applications and gems.

File: Gemfile [hunk 2]
```diff
@@ -4,11 +4,9 @@ source 'https://rubygems.org'
...
-  gem 'rubocop', '~> 1.21'
-  gem 'rubocop-rake', require: false
-  gem 'rubocop-rspec', require: false
-end
+gem 'mail'
+gem 'rake', '~> 13.0'
+gem 'rspec', '~> 3.0'
+gem 'rubocop', '~> 1.21'
+gem 'rubocop-rake'
+gem 'rubocop-rspec'
```
Comment (i7an) [inline Gemfile:12]: Dependencies are required explicitly. There is no need for groups or `require: false`.

**Review by leonid-shevtsov — APPROVED** [review]

**Review by mklocek — APPROVED** [review]

**Review by vittorius — APPROVED** [review]

**Pull request merged**